tools/validators/ontology_validator/yamlformat/validator/presubmit_validate_types_lib.py

```diff
@@ -62,15 +62,27 @@ class ConfigUniverse(findings_lib.Findings):
 
   def __init__(self, entity_type_universe, field_universe, subfield_universe,
                state_universe, unit_universe):
-    super(ConfigUniverse, self).__init__()
+    super().__init__()
     self.entity_type_universe = entity_type_universe
     self.field_universe = field_universe
     self.subfield_universe = subfield_universe
     self.state_universe = state_universe
     self.unit_universe = unit_universe
+    self.unit_universe_reverse_map = self._ArrangeUnitsByMeasurement()
     # temporary placeholder for instance validator
     self.connections_universe = None
 
+  def _ArrangeUnitsByMeasurement(self):
+    if not self.unit_universe:
+      print('UnitUniverse undefined in ConfigUniverse')
+      return None
+    
+    unitsByMeasurement = dict()
+    units = self.unit_universe.GetUnitsMap('')
```

Comment:
label the bug here.  The ontology technically supports units below the global namespace.  This does not support that behavior correctly.